ui/css/intermittent-failures.css

```diff
@@ -67,6 +67,7 @@ ul {
 
 .failure_li {
   max-width: 800px;
+  max-width: 76vw;
```

Comment:
vw doesn't seem to work on the latest version of Firefox, which is what we care about supporting. I think you should change the max-width to a little less than the max-width of the table.

Reply:
Oh, I should have test with Firefox too... :disappointed: 

Will that make it larger for wide screens? I mean reducing the max-width. :thinking:

Reply:
No, because you are going by the max-width of the table which doesn't get bigger after a certain screen size.

Reply:
Ok, thanks :smile: